### PR TITLE
Add release note for v1.6.6

### PR DESCRIPTION
**Pull request by yongtang**
Part of #3519 for v1.6.6 release.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: notes/coredns-1.6.6.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+# Security 
+
+The github.com/miekg/dns has been updated to v1.1.25 to fix a DNS related security
+vulnerability (https://github.com/miekg/dns/issues/1043).
+
+# Plugins
+
+A new plugin [*bufsize*](/plugins/bufsize) has been added to prevents IP fragmentation
```
Comment (stp-ip) [inline notes/coredns-1.6.6.md]:
```suggestion
A new plugin [*bufsize*](/plugins/bufsize) has been added that prevents IP fragmentation
```

File: notes/coredns-1.6.6.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+for the DNS Flag Day 2020 and to deal with DNS vulnerability.
```
Comment (stp-ip) [inline notes/coredns-1.6.6.md]:
```suggestion
for the DNS Flag Day 2020 and to deal with DNS vulnerabilities.
```

File: notes/coredns-1.6.6.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+* [*cache*](/plugin/cache) added a `serve_stale` option similar to `unbound`'s `serve_expired`.
+* [*sign*](/plugin/sign) fix signing of authoritative data that are not authoritative for.
```
Comment (stp-ip) [inline notes/coredns-1.6.6.md]:
```suggestion
* [*sign*](/plugin/sign) fix signing of authoritative data that we are not authoritative for.
```

**Review by stp-ip — CHANGES_REQUESTED** [review]

**Comment by miekg** [comment]
looks good.

I'll kick of a release tomorrow/this evening.

Thanks!

**Pull request merged**